### DISCUSSION TITLE
原民族語版ChatGPT

### DISCUSSION THREAD

**Discussion by xipeiguo** [1101-GS4719-C/Python-Community-GS4719-C#2092] [General]
https://www.chinatimes.com/realtimenews/20240902002728-260405?chdtv
我認為這是一項很有意義的開發，多元文化的保存和傳承是我們一直在乎且希望達成的，而原住民族群各自部落講的版本也或多或少有些許差異，比起期待原民族語版CHATGPT帶給人們的便利性，我還是更期待它對於文化的保存。

**Comment by Chattenoire** [comment]
從更大的脈絡下看，這是一場語言工具在地化的反西方解殖；殖民不只是歷史，而是正在進行式：從政經到知識論，乃至於到語言，都是一種西方認識論為中心的殖民。目前來說，只是從歐洲過渡到美洲，然後在近40年慢慢瓦解而已。

有興趣了解更多的話，可以參閱《解殖：全球殖民性與世界失序》該書，有詳盡的理論框架。
